packages/renderless/src/common/deps/useUserAgent.ts

```
@@ -0,0 +1,18 @@
+import { isServer } from './dom'
+
+function getIsIOS() {
+  if (isServer) return false
+  return (
+    window.navigator &&
+    window.navigator.userAgent &&
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)